### PR TITLE
[ISSUE #5354] [RIP-46] Implement proxy metrics framework

### PR DESCRIPTION
**Pull request by ShadowySpirits**
Please take a look. @lollipopjin @MatrixHB

### CODE REVIEW

File: broker/src/main/java/org/apache/rocketmq/broker/metrics/BrokerMetricsManager.java [hunk 1]
```diff
@@ -108,26 +113,26 @@ private boolean checkConfig() {
...
                 if (split.length != 2) {
-                    LOGGER.warn("brokerMetricsLabel is not valid: {}", brokerConfig.getBrokerMetricsLabel());
+                    LOGGER.warn("metricsLabel is not valid: {}", brokerConfig.getMetricsLabel());
                     continue;
                 }
                 LABEL_MAP.put(split[0], split[1]);
             }
         }
-        if (brokerConfig.isBrokerMetricsPreferDelta()) {
+        if (brokerConfig.isMetricsPreferDelta()) {
```
Comment (lizhanhui) [inline broker/src/main/java/org/apache/rocketmq/broker/metrics/BrokerMetricsManager.java:132]: isMetricsPreferDelta --> isDeltaMetricPerferred

Comment (ShadowySpirits) [reply broker/src/main/java/org/apache/rocketmq/broker/metrics/BrokerMetricsManager.java:132]: I want to keep all metrics related config prefixed with "metrics". Do you have any better suggestions?

Comment (lizhanhui) [reply broker/src/main/java/org/apache/rocketmq/broker/metrics/BrokerMetricsManager.java:132]: > I want to keep all metrics related config prefixed with "metrics". Do you have any better suggestions?

How about isMetricInDelta?

Comment (ShadowySpirits) [reply broker/src/main/java/org/apache/rocketmq/broker/metrics/BrokerMetricsManager.java]: > How about isMetricInDelta?
I have done this modification.

**Review by lizhanhui — APPROVED** [review]

**Pull request merged**